packages/wp-story-editor/src/api/media.js

```diff
@@ -59,6 +59,10 @@ export function getMedia(
     apiPath = addQueryArgs(apiPath, { cache_bust: true });
   }
 
+  if (include) {
+    apiPath = addQueryArgs(apiPath, { include });
```

Comment:
Is this still needed? `getPublisherLogos` returns all publisher logos + their URLs

Reply:
Nah, I just didn't revert that.